packages/paste-website/src/pages/layout/stack/index.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Stack - Layout
+description: Stack is a horizontal and vertical flow component used to create space between components.
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {Card} from '@twilio-paste/card';
+import {FormInput, FormLabel} from '@twilio-paste/form';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Stack} from '@twilio-paste/stack';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/stack/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteLayout(filter: {name: {eq: "@twilio-paste/stack"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/stack/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Stack"
+  categoryRoute={SidebarCategoryRoutes.LAYOUT}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/layout/stack"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/layout-stack--stack"
+  data={props.data.allPasteLayout.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Stack
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+It allows you to easily stack elements together with defined spacing between them.
+It can be set up either vertically or horizontally, and uses tokens for spacing.
+
+#### Accessibility
+
+By default, it has no accessibility concerns. If you use Stack as a custom element,
```

Comment:
"By default, Stack has no accessibility concerns. While using Stack, it is up to you to manage the resulting accessibility implications. You may want to consider how
the stack affects readability due to ordering, and the visual or content hierarchy
of your page."

Reply:
Replace with this

Reply:
Done.